pages/_/stats.js

```diff
@@ -0,0 +1,414 @@
+      "retrieval-status": 6,
+      errors: 0,
+    },
+    {
+      id: `file-12`,
+      icon: "PNG",
+      file: "example-painting-a-5.jpg",
+      miner: "Example Miner F",
+      "deal-cid": "GHREREFGvFoPwmQUSa92pxnxjQuPU",
+      date: Strings.toDate("2014-07-24 00:00:00 UTC"),
+      size: Strings.bytesToSize(44432, 3),
+      amount: Strings.formatAsFilecoin(20.13),
+      remaining: Strings.getRemainingTime(320021),
+      cid: "dfsffdbPFo2XbhXkhBvFoPwmQUSa92pxnxjQuPU",
+      "retrieval-status": 6,
+      errors: 0,
+    },
+    {
+      id: `file-13`,
+      icon: "PNG",
+      file: "pending-file-1.jpg",
+      miner: "Example Miner G",
+      "deal-cid": "13REREFGvFoPwmQUSa92pxnxjQuPU",
+      date: Strings.toDate("2014-07-24 00:00:00 UTC"),
+      size: Strings.bytesToSize(44432, 3),
+      amount: Strings.formatAsFilecoin(20.13),
+      remaining: null,
+      cid: "13sffdbPFo2XbhXkhBvFoPwmQUSa92pxnxjQuPU",
+      "retrieval-status": 1,
+    },
+    {
+      id: `file-14`,
+      icon: "PNG",
+      file: "pending-file-2.jpg",
+      miner: "Example Miner G",
+      "deal-cid": "14REREFGvFoPwmQUSa92pxnxjQuPU",
+      date: Strings.toDate("2014-07-24 00:00:00 UTC"),
+      size: Strings.bytesToSize(44432, 3),
+      amount: Strings.formatAsFilecoin(20.13),
+      remaining: null,
+      cid: "14sffdbPFo2XbhXkhBvFoPwmQUSa92pxnxjQuPU",
+      "retrieval-status": 2,
+    },
+    {
+      id: `file-15`,
+      icon: "PNG",
+      file: "pending-file-3.jpg",
+      miner: "Example Miner H",
+      "deal-cid": "15REREFGvFoPwmQUSa92pxnxjQuPU",
+      date: Strings.toDate("2014-07-24 00:00:00 UTC"),
+      size: Strings.bytesToSize(44432, 3),
+      amount: Strings.formatAsFilecoin(20.13),
+      remaining: null,
+      cid: "15sffdbPFo2XbhXkhBvFoPwmQUSa92pxnxjQuPU",
+      "retrieval-status": 3,
+    },
+    {
+      id: `file-16`,
+      icon: "PNG",
+      file: "pending-file-4.jpg",
+      miner: "Example Miner I",
+      "deal-cid": "16REREFGvFoPwmQUSa92pxnxjQuPU",
+      date: Strings.toDate("2014-07-24 00:00:00 UTC"),
+      size: Strings.bytesToSize(44432, 3),
+      amount: Strings.formatAsFilecoin(20.13),
+      remaining: null,
+      cid: "16sffdbPFo2XbhXkhBvFoPwmQUSa92pxnxjQuPU",
+      "retrieval-status": 4,
+    },
+    {
+      id: `file-17`,
+      icon: "PNG",
+      file: "pending-file-5.jpg",
+      miner: "Example Miner J",
+      "deal-cid": "17REREFGvFoPwmQUSa92pxnxjQuPU",
+      date: Strings.toDate("2014-07-24 00:00:00 UTC"),
+      size: Strings.bytesToSize(44432, 3),
+      amount: Strings.formatAsFilecoin(20.13),
+      remaining: null,
+      cid: "17sffdbPFo2XbhXkhBvFoPwmQUSa92pxnxjQuPU",
+      "retrieval-status": 5,
+    },
+  ],
+};
+
+export default class StatsPage extends React.Component {
+  render() {
+    const title = `Stats Page`;
+    const description = "This is to experiment with analytics.";
+    const url = "https://slate.host/analytics";
+
+    return (
+      <ScenePage>
```

Comment:
This is the wrong component, because its not a scene, please look at `/pages/_/test.js` if you want to make a test page. Or take my previous comment and make it an experience page.

Reply:
Revised to match formatting of test page.